public_html/components/com_jed/views/extension/tmpl/default.php

```diff
@@ -6,208 +6,229 @@
  * @license    GNU General Public License version 2 or later; see LICENSE.txt
  */
 
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
 use Joomla\CMS\Layout\LayoutHelper;
+use Joomla\CMS\Router\Route;
 
 defined('_JEXEC') or die;
 
 /** @var JedViewExtension $this */
 ?>
 
-<div class="jed-wrapper jed-extension-header">
-	<div class="jed-container">
-		<div class="jed-extension-header__content">
-			<div class="jed-extension-header__item">
-				<h1 class="heading heading--xl">PWT Image</h1>
-				<div class="jed-extension-header__info">
-					<div class="jed-extension-header__developer">By <a href="#">Perfect Web Team</a></div>
-					<div class="stars-wrapper">
-						<div class="stars">
-							<div class="star"><span aria-hidden="true" class="icon-star"></span></div>
-							<div class="star"><span aria-hidden="true" class="icon-star"></span></div>
-							<div class="star"><span aria-hidden="true" class="icon-star"></span></div>
-							<div class="star"><span aria-hidden="true" class="icon-star"></span></div>
-							<div class="star"><span aria-hidden="true" class="icon-star-empty"></span></div>
+	<div class="jed-wrapper jed-extension-header">
+		<div class="jed-container">
+			<div class="jed-extension-header__content">
+				<div class="jed-extension-header__item">
+					<h1 class="heading heading--xl"><?php echo $this->item->title; ?></h1>
+					<div class="jed-extension-header__info">
+						<div class="jed-extension-header__developer">By
+							<a href="#"><?php echo $this->item->developer; ?></a></div>
+						<div class="stars-wrapper">
+							<?php echo LayoutHelper::render('elements.stars', ['score' => $this->item->score]); ?>
+							<a href="#reviews"><?php echo $this->item->reviews; ?> reviews</a>
 						</div>
-						<a href="#">132 reviews</a>
 					</div>
 				</div>
-			</div>
-			<div class="jed-extension-header__item">
-				<div class="jed-extension-header__buttons button-group">
-					<a href="#" class="button button--blue button--big">Get extension</a>
-					<a href="#" class="button button--blue button--big button--icon-only">
-						<span aria-hidden="true" class="icon-heart-2"></span>
-						<span class="visually-hidden">Favourite</span>
-					</a>
+				<div class="jed-extension-header__item">
+					<div class="jed-extension-header__buttons button-group">
+						<?php if ($this->item->homepageLink): ?>
+							<a href="<?php echo $this->item->homepageLink; ?>" class="button button--blue button--big">Get extension</a>
```

Comment:
Change to Download Link

Reply:
If I recall correctly we discussed this during the last sprint, and that we would change it to the extension homepage instead. Happy to change it to the download URL, but in that case, we don't have a link to the extension homepage :)

Reply:
I remember debating about it, but can't remember the result of the conversation. 

I vote to change it to Download Link for the reason that developers will put the link to their homepage in there, this doesn't make sense if the button says Get Extension and it takes you to a page where you then have to navigate to get to the extension in order to download it. 

Where you have Download Link now, change that to Homepage Link, then we aren't lossing any links, and the Download Link will have the most prominent position at the top of the page.

Reply:
Changed the links!